connect/src/com/telenor/connect/id/ConnectIdService.java

```diff
@@ -36,7 +36,8 @@ public ConnectIdService(
 
     public void getValidAccessToken(final AccessTokenCallback callback) {
         if (retrieveTokens() == null) {
-            throw new ConnectRefreshTokenMissingException("Failed to call retrieveTokens().");
+            throw new ConnectRefreshTokenMissingException("retrieveTokens() returned null. Tokens" +
+                    "are missing.");
```

Comment:
Missing space. Consider inserting the newline before the string literal instead, so that you do not need to split it up. I still think it is confusing to throw "ConnectRefreshTokenMissingException" when the client is trying to get an access token